_meta/beat.yml

```diff
@@ -414,6 +414,241 @@ output.elasticsearch:
   # Pretty print json event
   #pretty: false
 
+#----------------------------- Logstash output ---------------------------------
+#output.logstash:
+  # Boolean flag to enable or disable the output module.
+  #enabled: true
```

Comment:
this is not enabled by default, right? (same for kafka bellow)

Reply:
good point, fixed